js/email-validation.js

```diff
@@ -1,14 +1,14 @@
 const form = document.getElementById('contact-form');
 const email = document.getElementById('email');
-const errorDisplay = document.getElementById('errorDisplay')
+const errorDisplay = document.getElementById('errorDisplay');
 const checker = /^[a-z0-9._-]+@[a-z0-9.-]+\.[a-z]{2,4}$/g;
 
 form.addEventListener('submit', (e) => {
     if (!email.value.match(checker)) {
         e.preventDefault();
         errorDisplay.style.visibility = 'visible';
         errorDisplay.classList.add('error-message');
-        errorDisplay.textContent = '*Your email address should be all in lowercase'
+        errorDisplay.textContent = '*Your email address should be all be in lowercase';
     } else {
         errorDisplay.style.visibility = 'hidden';
     }
```

Comment:
- [ ] `required`
- [ ] Amazing job so far 👏 👏 
- [ ] when i click on the `send` button i'm supposed to get a `success message` like the screenshot please kindly make adjustments. see the [form spree](https://formspree.io/) for configuration
![image](https://user-images.githubusercontent.com/66230306/181474434-129ec892-5fe5-4549-aecd-5b7b6a2ed172.png)